.github/workflows/ci.yml

```
@@ -32,7 +32,9 @@ jobs:
       - name: Build e2e
         working-directory: ./test/sdk-package-test
         run: yarn run build
+        env:
+          PRIVATE_KEY: ${{ secrets.PRIVATE_KEY }}
 
       - name: Run e2e tests
         working-directory: ./test/sdk-package-test
-        run: yarn run test:e2e
\ No newline at end of file
+        run: yarn run test:e2e
```

probably need the env here?
```suggestion
        run: yarn run test:e2e
        env:
          PRIVATE_KEY: ${{ secrets.PRIVATE_KEY }}
```